Microsoft.AAD/DomainServices Swagger: Add new api-version 2017-06-01

jihochang opened the pull request:
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

jihochang committed:
Microsoft.AAD/DomainServices swagger spec and examples

jihochang committed:
Microsoft.AAD/DomainServices: Add new api-version 2017-06-01

msftclas commented:
@jihochang,
Thanks for your contribution as a Microsoft full-time employee or intern. You do not need to sign a CLA.
_Thanks,
Microsoft Pull Request Bot_

jihochang commented:
Previous Pull Request accepted by veronicagg

https://github.com/Azure/azure-rest-api-specs/pull/1689#issuecomment-330640661

samedder committed:
Adding 6.0 ServiceFabric specification (#1732)

* Duplicating 5.6 swagger for 6.0 in Service Fabric

* Updating 6.0 to correct spec

* Removing negative status code responses

* Updating ReadMe

* Removing host

* Revert "Removing host"

This reverts commit 689b93f01569d84355e086a98b6042ba1a6bfedf.

Commit:
EventHub: updated for GEO DR  (#1709)

* Removed maxLength from entity name

* Geo DR changes

* review changes

* alias connection string added.

* description updated

* Review comments and examples

* pluralize opertionsIds and reveted delete response code to 200

* Removed LastSync Property

sasankv-msft committed:
Adding swagger coverage for 'PerformanceTiers' and 'CheckNameAvailability' (#1733)

operations for PostgreSQL and MySQL (API version 2017-04-30)

Added coverage for the following operations:
* PerformanceTiersList
* LocationBasedPerformanceTierList
* CheckNameAvailability

sergey-shandar commented:
@ravbhatnagar new API version `DomainServicesSwagger.json`

Commit:
Servicebus: updated for GEO DR (#1714)

* GEO DR related changes

* Review comments and examples

* Geo DR - Pluralize opertionsIds and reveted delete response code to 200

* Removed LastSync Property

juhacket committed:
Release ServiceFabric 2017-07-01-preview swagger specification (#1717)

* Release ServiceFabric 2017-07-01-preview swagger specification

This change introduces applicationType, version, application, and service ARM resources for the ServiceFabric resource provider.

* Release ServiceFabric 2017-07-01-preview swagger specification

This change introduces applicationType, version, application, and service ARM resources for the ServiceFabric resource provider.

olydis committed:
Update readme.md

juhacket committed:
Remove incorrect title from readme.md (#1748)

* Release ServiceFabric 2017-07-01-preview swagger specification

This change introduces applicationType, version, application, and service ARM resources for the ServiceFabric resource provider.

* Release ServiceFabric 2017-07-01-preview swagger specification

This change introduces applicationType, version, application, and service ARM resources for the ServiceFabric resource provider.

* Remove incorrect title from readme.md

* Remove incorrect title from readme.md

jihochang commented:
The api-version itself is not new. We are just adding the swagger for it now.

olydis committed:
Update readme.md

olydis committed:
file name fix

olydis commented on specification/domainservices/resource-manager/readme.md:
````diff
@@ -39,6 +39,14 @@ input-file:
+
+``` yaml $(tag) == 'package-2017-06'
+input-file:
+- Microsoft.AAD/2017-06-01/domainservices.json
````
note that this path is *wrong*, the filename is `DomainServicesSwagger.json`. I've just fixed that for the previous API version, please make sure you also adjust this accordingly.

solankisamir committed:
Fix issue with Equivalent_PATH error with apimreports.json (#1750)

* Add Query to Export API and nit documentation update

* Fixed Parameter name

* Fixed Parameter name

* Update description to include setting to disable TLS

* Update information for cipher.

* Fix spelling of cipher

* Fix EQUIVALENT_PATH error with apimreports.json

ro-joowan committed:
Scope2 (#1745)

* [ADLA] - Adding Scope APIs/properties

* [ADLA/S] - [BUGFIX] - Adding missed properties to provisioningState

* [ADLA] - Revising the description for adlaJobDnsSuffixInPath

* [ADLA] - [CODE] - Adding "notifier" field to ScopeJobProperties and CreateScopeJobProperties

jihochang committed:
Fix readme swagger file path

jihochang commented:
I renamed the swagger file to match what is in the readme file.

matthchr committed:
Add links to filter details for supporting APIs (#1756)

Commit:
reintroducing ecc-support changes from msfcolombo (#1724)

Commit:
reordering JsonWebKey properties to ensure generated code is backwards compatible (#1762)

olydis committed:
Update .travis.yml

olydis committed:
bump (#1766)

jihochang commented:
Could you take another look and approve the pull request? Our ARM swagger deadline for this is tomorrow. This is an already existing API-VERSION and we are just now adding the swagger files corresponding to it.

Commit:
Removing AzureRegion parameterization since Search APIs dont need it (#1768)

veronicagg committed:
small update to message from linter CI (#1769)

djyou committed:
Update SKU (#1771)

mastermanu committed:
IoT Hub - X509CA support  (#1723)

* Initial checkin

* Fix syntax error

* Adds id and type to certificate response structure as well

* address reviewer comments

* fixes typo

* Adds limit on certificate name

* Fixes nesting of etag, name, id properties

* adds property reference for certificate description

vivsriaus committed:
Add support for policy at ManagementGroup level (#1754)

* Add support for policy at ManagementGroup level

* Add "x-ms-parameter-location": "method" for ManagementGroupId parameter

jihochang commented:
Ping any update on this?

Commit:
BatchAI. Initial drop of REST API spec for Batch AI. (#1772)

* BatchAI. Initial drop of REST API spec for Batch AI.

* BatchAI. Fixed samples for leaving node count

* BatchAI. Fixed examples and a typo in spec

* BatchAI. Fixed FileServerStatus to generate valid c# comment.

* BatchAI. Removed unimplemented functionality from file servers.

vishrutshah committed:
[Travis CI] Using autorest 2.0.4152 to get bug fix on reporting on mem:/// instead of file:/// (#1775)

* Using autorest 2.0.4152 to get bug fix on reporting on mem:/// instead of file:///

* Updating features file to test the CI

* Revert "Updating features file to test the CI"

This reverts commit c2159101f7d84c608f9908388513a23ee0f41c5f.

ravbhatnagar commented:
Looks good.

ajinkyapotdarvjti committed:
Made sourceAddressPrefix and destinationAddressPrefix non-mandatory params (#1757)

Commit:
BatchAI. Getting rid of legacy "learning" brand. (#1784)

Originally the service was called "Batch AI learning service", now it's
just "Batch AI". So, we need to get rid of legacy "learning" everywhere.

sergey-shandar commented on specification/domainservices/resource-manager/Microsoft.AAD/2017-01-01/domainservices.json:
```diff
@@ -26,24 +26,30 @@
+            "description": "The operation succeeded. The response contains the possible operations on the Microsoft.AAD resource provider.",
             "schema": {
-              "$ref": "#/definitions/OperationEntityListResult"
+              "$ref": "#/definitions/OperationListResult"
```
This is definitely an SDK breaking changes. Is there a significant reason to do these breaking changes in the old version of API?

milantomic6 committed:
Cancel pipeline run api (#1782)

* Cancel pipeline run api

* CR issues

jihochang committed:
Microsoft.AAD/DomainServices Swagger: Add new api-version 2017-06-01

jihochang committed:
Merge branch 'DomainServices' of https://github.com/jihochang/azure-rest-api-specs into DomainServices

jihochang committed:
Merge: Resolve Conflicts

jihochang commented:
Creating new pull request, due to bad merge. 

New Pull Request: https://github.com/Azure/azure-rest-api-specs/pull/1790
